src/Compilers/Test/Utilities/VisualBasic/Extensions.vb

```diff
@@ -65,7 +65,7 @@ Friend Module Extensions
         Dim lastContainer As NamespaceOrTypeSymbol = Nothing
         Dim members = GetMembers(container, qualifiedName, lastContainer)
         If members.Length = 0 Then
-            Assert.True(False, "Available members:" & vbCrLf + String.Join(vbCrLf, lastContainer.GetMembers()))
+            Return Nothing
```

Comment:
>Return Nothing [](http://example.com/codeflow?start=12&length=14)

Why remove this? Seems useful.

Reply:
Removing the assert means this now matches what the C# side does. For deletes we expect `c.GetMember("C.Goo")` to return null, and the helpers allow for it, but VB was failing the test because of this.